iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingViewIndex.java

```diff
@@ -243,17 +269,22 @@ Identifiable getIdentifiable(final Identifiable identifiable) {
     }
 
     Collection<Line> getLines() {
-        // Search Line into merging & working networks
-        return getNetworkStream()
-                .flatMap(Network::getLineStream)
-                .map(this::getLine)
+        // Search Line into merging & working networks, and MergedLines
+        return Stream.concat(getNetworkStream().flatMap(Network::getLineStream)
+                        .map(this::getLine),
+                mergedLineCached.values().stream())
                 .collect(Collectors.toList());
     }
 
+    boolean asDanglingLine(final DanglingLine dl) {
```

Comment:
See above: maybe you should rename this method to `isMerged()`

Reply:
ok